spec/run_oc_pedant.rb

```diff
@@ -43,7 +43,15 @@
     '--skip-omnibus',
     '--skip-usags',
     '--exclude-internal-orgs',
-    '--skip-headers'
+    '--skip-headers',
+
+    # Chef 12 features not yet 100% supported by Chef Zero
```

Comment:
Agreed - api-v1 is actually pretty critical as well and the implied API versioning support I've opened that as #154